conf/adcm/python_scripts/run_ldap_sync.py

```diff
@@ -20,7 +20,14 @@
 
 import adcm.init_django  # pylint: disable=unused-import
 from rbac.models import User, Group, OriginType
-from rbac.ldap import _get_ldap_default_settings, configure_tls, is_tls
+from rbac.ldap import (
+    _get_groups_by_user_dn,
```

Comment:
Maybe better to make functions public as well as we import them in other modules (here)

Reply:
done